README.md

```diff
@@ -1,3 +1,10 @@
 # Mobile Security Guide Website repo
 
-This repo has the files for http://www2.owasp.org/www-project-mobile-security-testing-guide/. Please make sure you create a PR for changes. More to come soon.
\ No newline at end of file
+This repo has the files for https://owasp.org/www-project-mobile-security-testing-guide/. 
+
+A few things to keep in mind:
+
+ - For changes: file a [PR](https://github.com/OWASP/www-project-mobile-security-testing-guide/pulls) (Fork this repo, create changes on your local fork and then do a PR, like described in [the MSTG](https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md#how-to-set-up-your-contributor-environment), just make sure you use the right url (github.com/OWASP/www-project-mobile-security-testing-guide instead of github.com/OWASP/owasp-mstg/)
```

Comment:
```suggestion
 - For changes: file a [PR](https://github.com/OWASP/www-project-mobile-security-testing-guide/pulls) by forking this repo, creating changes on your local fork and then opening a PR, like described in the [Contributing page](https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md#how-to-set-up-your-contributor-environment) in the MSTG repo. Please make sure you use the right URL ("github.com/OWASP/www-project-mobile-security-testing-guide" instead of "github.com/OWASP/owasp-mstg").
```